src/tests/BuildWasmApps/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -133,6 +133,7 @@ public BuildTestBase(ITestOutputHelper output, SharedBuildPerTestClassFixture bu
             (string testCommand, string extraXHarnessArgs) = host switch
             {
                 RunHost.V8 => ("wasm test", "--js-file=runtime.js --engine=V8 -v trace"),
+                // RunHost.NodeJS => ("wasm test", "--js-file=runtime.js --engine=NodeJS -v trace"),
```

Comment:
This will be uncommented once Ankit's Wasm.Build.Tests for Windows PR lands. See other comment for explanation about the test failures currently.

Reply:
@radical I'm not sure about context here, could you please give me some hint ?